pkg/storage/blobstoreutil/blob_store_util.go

```diff
@@ -0,0 +1,44 @@
+package blobstoreutil
```

Comment:
please something like storage/module/upload.go
so we dont end up with `util`packages